lib/map.jsx

```diff
@@ -0,0 +1,28 @@
+var React = require('react');
+
+var MapComponent = React.createClass({
+
+    propTypes: {
+      accessToken: React.PropTypes.string.isRequired,
+      mapId: React.PropTypes.string.isRequired,
+    },
+
+    componentDidMount: function () {
+      console.log('here1')
+      // Provide your access token
+      L.mapbox.accessToken = this.props.accessToken;
+
+      // Create a map in the div #map
+      var map = L.mapbox.map('map', this.props.mapId);
+    },
+
+    // Called on initialization and after each change to the components
+    // props or state
+    render: function () {
+      return (
+        <div id="map"></div>
```

Comment:
Hmm, so a downside of this is that we're using a global ID for the map here... While that's not altogether bad, since our current design only puts one map on the page, the assumption is generally that React components can be instantiated multiple times on the page. I recommend either changing the name of this component to `SingletonMapComponent`, to indicate that it's a singleton, or removing the `id` and instead passing an element reference (via `this.getDOMNode()`) into the [`L.mapbox.map`](https://www.mapbox.com/mapbox.js/api/v2.1.5/l-mapbox-map/) constructor.